base/shell/rshell/CQuickLaunchBand.h

```diff
@@ -1,9 +1,8 @@
 /*
  * PROJECT:     ReactOS shell extensions
- * LICENSE:     GPL - See COPYING in the top level directory
- * FILE:        dll/shellext/qcklnch/CQuickLaunchBand.h
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
```

Comment:
does it make sense to offer some of the rshell files with GPL2.0 and some like misc.cpp with LGPL2.1? (I don't really care about that myself, but some other people might care)